### PR TITLE
Dogfooding `Annotate new data` with some NLU annotation #1

### PR DESCRIPTION
**Pull request by TyDunn**

### CODE REVIEW

File: data/nlu/nlu.md [hunk 1]
```diff
@@ -4913,22 +4938,21 @@
 - thnks
 - thx
 - yes thanks
+- thanks for your information
 
-## lookup: location
-data/nlu/lookups/location.txt
```
Comment (akelad) [inline data/nlu/nlu.md]: @ArjaanBuijk can you re-add the lookup tables on Rasa X? Not sure why these got deleted

Comment (ArjaanBuijk) [reply data/nlu/nlu.md]: @akelad , I will look into this and fix it.

File: data/nlu/nlu.md [hunk 2]
```diff
@@ -3466,6 +3479,8 @@
 - you're the most moronic person I know
 - your mom gay
 - you are annoying
+- don't you have a sweater
```
Comment (erohmensing) [inline data/nlu/nlu.md:3482]: @TyDunn is this actually an insult? 😂

File: data/nlu/nlu.md [hunk 3]
```diff
@@ -4421,6 +4441,11 @@
 - you will know it from the single red rose it carries between its teeth
 - you're a woman
 - ı am learning python
+- CALL THE POLICE
+- I want to speak binary with you
```
Comment (erohmensing) [inline data/nlu/nlu.md:4445]: should this be in `ask_languagesbot`? Would kindof be a perfect response haha

Comment (TyDunn) [reply data/nlu/nlu.md:4445]: Yes 👍

Comment (TyDunn) [inline data/nlu/nlu.md:4445]:
```suggestion
```

File: data/nlu/nlu.md (hunk 2 above)
Comment (TyDunn) [reply data/nlu/nlu.md:3483]: PTSD from people always calling me out for "not being dressed properly"

Comment (erohmensing) [reply data/nlu/nlu.md:3483]: i think OOS for that

Comment (TyDunn) [inline data/nlu/nlu.md:3483]:
```suggestion
```

**Pull request merged**